src/components/tree-item/ch-tree-item.tsx

```diff
@@ -550,6 +550,14 @@ export class ChTreeItem {
         this.checkboxClickedEvent.emit(true);
       }
     }
+    if (treeRef.toggleCheckboxes) {
```

Comment:
Summary of changes:
 - Use semantic names when defining variables. For example, `treeItems` instead of `children`.

 - StencilJS (and probably TypeScript in general) inferes types -if posible- when using querySelector. So, `item` should already be typed.
 Also, don't use `item = item as type` to force type inference.


 - `console.log` should not be sent to production code. In any case, this issue is mine, since this rule can be set in the Linter.

 - Do not use `condition && doSomething` instead of if statements.